docs/link-reference-definitions.md

```diff
@@ -68,6 +68,7 @@ See [[link-reference-definition-improvements]] for further discussion on current
 
 
 [//begin]: # "Autogenerated link references for markdown compatibility"
-[workspace-janitor]: workspace-janitor.md "Janitor"
-[link-reference-definition-improvements]: link-reference-definition-improvements.md "Link Reference Definition Improvements"
+[workspace-janitor]: workspace-janitor "Janitor"
```

Comment:
these links need to include the `.md` extension, you need to change the setting for the wikilink definitions to `withExtension`

Reply:
How do I make this work? The setting is already there in `docs/.vscode`, but doesn't seem to work.

Reply:
you need to be sure that:
- you have opened that (and not the parent folder, which has its own `.vscode`, in which case check the setting there
- you don't have a version of the foam extension that is too old (which doesn't support the setting)

let me know if that fixes it

Reply:
I see, you have to be in the workspace